dockerplugin.py

```diff
@@ -90,22 +90,34 @@ def read_blkio_stats(container, dimensions, stats, t):
         # numbers). We need to group and report the stats of each block
         # device independently.
         blkio_stats = {}
+        blkio_major_stats = {}
+        blkio_minor_stats = {}
+
         for value in values:
+
             k = '{key}-{major}-{minor}'.format(key=key,
```

Comment:
i'll check if we should change k.  I left it intact as it looked like it was still needed.